masakari-controller/controller/masakari_config.py

```diff
@@ -179,9 +267,15 @@ def _log_setup(self):
 
         level = self.conf_log.get('log_level')
 
+        logging_context_format_string = '%(asctime)s.%(msecs)03d %(process)d %(levelname)s ' \
+            + '%(name)s [%(thread_name)s %(record_identifier)s] %(message)s'
         logging.set_defaults(
+            logging_context_format_string=logging_context_format_string,
             default_log_levels=logging.get_default_log_levels() +
             ['controller=' + level])
+        RequestContext(thread_name=str(
```

Comment:
There is two reasons:

1. 
I'd like to make RequestContext holding thread_name and record_identifier for logging.
So I made the class that is inherited oslo_context.
(RequestContext in oslo_context does not have instance variables of thread_name/record_identifier.)

2.
get_current() of "oslo_context/context.py" will be called when the class that is inherited oslo_context outputs logs.
The return value of get_current() is the "_request_store" object that is defined in "oslo_context/context.py" .
That is not the "_request_store" of the class that is inherited oslo_context.
So I cannot outputs thread_name and record_identifier for logs if I'll just make the class that is inherited oslo_context.
I overwrite _request_store of oslo_context in masakari_config.py for that reason.